src/main/resources/templates/fragments/navbar.html

```diff
@@ -0,0 +1,32 @@
+<header th:fragment="navbar" id="navbar" aria-label="Main navigation">
+    <nav class="navbar" role="navigation">
+        <div class="navbar-brand">
+            <a class="navbar-item" href="/">OurShop</a>
+        </div>
+
+            <!-- Dynamically generated navigation links -->
+            <div class="navbar-links">
+                <a th:each="link : ${navLinks}"
```

Comment:
For accessibility improvements consider wrapping around the `<a>` tag with `<li>` tag inside an `<ul>` tag

Reply:
tried it, but it looked bad. maybe when bulma is implemented